drafts/versions/02/draft-ietf-ippm-ioam-data-02.txt

```diff
@@ -1070,15 +1070,64 @@ Internet-Draft           In-situ OAM Data Fields           February 2018
 
    IOAM proof of transit option header:
 
-    0 1 2 3 4 5 6 7
-   +-+-+-+-+-+-+-+-+
-   |IOAM POT Type|P|
-   +-+-+-+-+-+-+-+-+
+    0                   1                   2                   3
+    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+   |IOAM POT Type  | IOAM POT flags|           Reserved            |
+   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+   IOAM proof of transit option data MUST be 4-octet aligned.:
 
-   IOAM proof of transit option data MUST be 4-octet aligned:
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |       POT Option data field determined by IOAM-POT-Type       ~
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+
+
+   IOAM POT Type:  8-bit identifier of a particular POT variant that
+      dictates the POT data that is included.  This document defines POT
+      Type 0:
+
+      0: POT data is a 16 Octet field as described below.
```

Comment:
It is a little awkward that this does not have a name. Not that I have any suggestions for that name ...